logger/proto/logger_config.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package logger;
+
+option go_package = "github.com/pokt-network/pocket/logger";
+
+message LoggerConfig {
+  LogLevel level = 1;
+  LogFormat format = 2;
+}
+
+enum LogLevel {
```

Comment:
I thought it's better to utilize enum since we know what potential values can be.

Reply:
> Still keeping enums because they can be useful for config validation.

If we aren't going to be using them, we should remove it.

> with the new config management via runtimeMgr I don't think I can use protobof types anymore.

Could you explain why? Let's see if we can update runtimeMgr for it.

Reply:
> Could you explain why? Let's see if we can update runtimeMgr for it.

Sure! So if we were to utilize enum types in the *Config types, this would trigger an import cycle, here's an example:
https://github.com/okdas/pocket/commit/3d83f223aec8ed96521ebe6da28262238bb565b4

Please let me know if you think of a better way of utilizing enums! :)

Reply:
Got it. Keep the enum encapsulated to the `shared` repo and use a string in shared.

In shared, you can add a comment that it is an enum that's being externally represented as a string due to cyclic dependencies and that we'll look into a way to improve it in the future.

Reply:
@Olshansk this is the part I've got most of my confusion from.
The reason I've suggested to use enums here for is because we know potential values for the configuration - and they cannot be any different from what we specify in enum. Think of this as of values validation/verification. If we specify string here, and we want to continue to use enums, I need to add a validation step on create to verify string value against our enums.

Reply:
Discussed offline. We will go with an enum + mapper approach and "let it fail" if the logging config is incorrect, which is valid.